onnxruntime/test/optimizer/runtime_optimization/graph_runtime_optimization_test.cc

```diff
@@ -264,6 +270,54 @@ TEST(GraphRuntimeOptimizationTest, QDQConv) {
       });
 }
 
+// if level 3 optimizations are enabled the NHWC transformer should convert the QLinearConv nodes to use channels_last
+void CheckNhwcTransformerIsApplied() {
```

Comment:
>CheckNhwcTransformerIsApplied [](http://example.com/codeflow?start=5&length=29)

nit: could this function be located closer to `TEST(GraphRuntimeOptimizationTest, TestNhwcTransformer)`? and be static/in the unnamed namespace?

Reply:
Moved the function back (was in that namespace but dropped out accidentally when resolving a merge conflict). 

Not sure it can be closer though given the QDQConv and ConvActivation tests both use SaveAndLoadRuntimeOptimizationsForModel which is defined in the anon namespace as well.